scripts/catalog-tests/generate-tdr-snapshot-metadata.py

```diff
@@ -0,0 +1,85 @@
+#------------------------------------------------------------------------------
+# Generate TDR Snapshot Metadata
+#
+# This script adds snapshots to the catalog as the test user from DC-218,
+# pulling the metadata from the resources JSON file
+# 
+# Before Running:
+#   python3 -m pip install requests argparse
+#   
+#   Authorization: Access the password in the vault using the appropriate command:
```

Comment:
How is the vault password used in the script?

Reply:
It's not, unfortunately - I couldn't figure out how to do it in a way that would be completely contained in the script. The commands to gcloud will open a browser window and prompt you to log in to that user.

Reply:
Oh I see. This is fine 👍